lib/features/composer/presentation/mixin/ai_scribe_in_composer_mixin.dart

```diff
@@ -16,6 +19,20 @@ mixin AIScribeInComposerMixin {
   RichTextMobileTabletController? get richTextMobileTabletController;
   ImagePaths get imagePaths;
   GenerateAITextInteractor get generateAITextInteractor;
+  Session? get session;
+  AccountId? get accountId;
+
+  bool get isAIScribeAvailable {
+    if (!AIConfig.isAiEnabled) return false;
+
```

Comment:
```suggestion
if (PlatformInfo.isMobile) return false;
```

Reply:
Added check related to mobile here 435f0ad4cedc2a6be9a0a28acb8a4c3078eb7f77